### PR TITLE
fix price 0 as free

### LINKED ISSUE #27: if price == 0, show "free" instead of Rs. 0
image

### PR DESCRIPTION
**Pull request by v27u27**
fixes #27 
I have also added '₹' symbol instead of 'Rs.' which was being used earlier, i hope it's okay.
Here's the screenshot from the build :- 

fix

### CODE REVIEW

File: app/helpers/format-price.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { helper } from '@ember/component/helper';
+
+export function formatPrice(params/*, hash*/) {
+	let price = parseInt(params[0]);
+	if(price == 0){
```
Comment (abhishek97) [inline app/helpers/format-price.js]: Just check for `NaN` here.

Comment (v27u27) [reply app/helpers/format-price.js]: sure, what should i return in case for NaN?

Comment (championswimmer) [reply app/helpers/format-price.js]: If there is no price, consider that as 0

Comment (championswimmer) [reply app/helpers/format-price.js]: And thus free

**Comment by v27u27** [comment]
@abhishek97 @championswimmer I've added the NaN check, please have a look.

**Comment by abhishek97** [comment]
Nice work. :+1:

**Comment by v27u27** [comment]
@abhishek97 thanks :+1:

**Pull request merged**